src/pages/login.tsx

```diff
@@ -0,0 +1,171 @@
+          <Box
+            p={8}
+            w="100%"
+            maxWidth="768px"
+            borderWidth={1}
+            borderRadius={16}
+            boxShadow="lg"
+            _hover={{ boxShadow: '2xl' }}
+            transition="all 1s"
+          >
+            <form
+              action="/api/auth/signin"
+              method="post"
+              onSubmit={handleSubmit}
+            >
+              <Flex flexDirection="column" alignItems="start">
+                <Heading
+                  as="h1"
+                  size="2xl"
+                  mb={8}
+                  textAlign="center"
+                  alignSelf="center"
+                >
+                  Login
+                </Heading>
+                {isVisible && (
+                  <Alert status="error" borderRadius={10} mb={8}>
+                    <AlertIcon />
+                    <AlertTitle>{errorMessage}!</AlertTitle>
+                    <AlertDescription>
+                      Remember not to enter &quot;@u.nus.edu&quot; yourself
+                    </AlertDescription>
+                    <CloseButton
+                      position="absolute"
+                      right={1}
+                      top={2}
+                      onClick={onClose}
+                    />
+                  </Alert>
+                )}
+                <FormControl>
+                  <FormLabel>NUS Email</FormLabel>
+                  <InputGroup>
+                    <Input
+                      type="text"
+                      value={userInfo.email}
+                      onChange={(e) =>
+                        setUserInfo({ ...userInfo, email: e.target.value })
+                      }
+                    />
+                    <InputRightAddon>@u.nus.edu</InputRightAddon>
+                  </InputGroup>
+                </FormControl>
+                <FormControl mt={4}>
+                  <FormLabel>Password</FormLabel>
+                  <InputGroup>
+                    <Input
+                      type={show ? 'text' : 'password'}
+                      value={userInfo.password}
+                      onChange={(e) =>
+                        setUserInfo({ ...userInfo, password: e.target.value })
+                      }
+                    />
+                    <InputRightElement width="4.5rem">
+                      <Button size="sm" onClick={handleShowPassword}>
+                        {show ? 'Hide' : 'Show'}
+                      </Button>
+                    </InputRightElement>
+                  </InputGroup>
+                </FormControl>
+                <ChakraNextLink
+                  href="/forgot-password"
+                  mt={4}
+                  textDecoration={'underline'}
+                >
+                  Forgot your password?
+                </ChakraNextLink>
+                <Button
+                  mt={8}
+                  size="lg"
+                  isLoading={false}
```

Comment:
Out of curiosity too, why did you write a `isLoading={false}`? Do you want to have a useState variable that can trigger the isLoading on and off based on the conditions?

Reply:
This was one of the Chakra Button options to potentially add a loading state. Will remove for now since it's unused.